pdns/dnsdist-protobuf.hh

```diff
@@ -123,9 +127,9 @@ class ProtoBufMetaKey
 
   struct KeyTypeDescription
   {
-    const std::string d_name;
-    const Type d_type;
-    const std::function<std::vector<std::string>(const DNSQuestion&, const std::string&, uint8_t)> d_func;
+    std::string d_name;
+    Type d_type;
+    std::function<std::vector<std::string>(const DNSQuestion&, const std::string&, uint8_t)> d_func;
```

Comment:
OK, now I'm getting it, you are removing `const` from members because of clang-tidy: https://clang.llvm.org/extra/clang-tidy/checks/cppcoreguidelines/avoid-const-or-ref-data-members.html
This is a very opinionated rule, and I'm not really convinced so I guess we need to discuss it with the rest of the team.